js-foundations.js

```diff
@@ -3,10 +3,31 @@
 
 // 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
 
-
+var inputNumber = -7
+const divByThree = (inputNumber) => {
+  if (inputNumber % 3 === 0) {
+    return inputNumber + ' is divisible by three'
+  }
+  else if (inputNumber % 3 !== 0){
+    return inputNumber + ' is not divisible by three'
+  }
+  else {
+    return 'Unexpected Input'
+  }
+}
+
+console.log(divByThree(inputNumber))
 
 // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
 
+var helloMe = { firstName: "Ian", lastName: "Thompson", age: 34 ,sex: "male" }
+const outputInfo = (helloMe) => {
+  let sentence = helloMe.firstName + ' ' + helloMe.lastName +' is a ' + helloMe.age + ' year old ' + helloMe.sex
+  return sentence
+}
+
+console.log(outputInfo(helloMe))
+
 
```

Comment:
This would be a great place to use string interpolation to make your code a bit cleaner.